src/main/resources/l10n/JabRef_da.properties

```diff
@@ -503,6 +503,9 @@ File_directory_is_not_set_or_does_not_exist\!=Filbiblioteket_er_ikke_sat_eller_e
 
 File_exists=Filen_eksisterer
 
+File_already_exists=
+File_'%0'_already_exists.=
```

Comment:
We have `'%0'_exists._Overwrite_file?` in JabRef. Maybe, the code can be adapted to allow file overwrite instead of just cancelling the request?